package.json

```diff
@@ -231,13 +231,13 @@
     "npm": ">=3.x"
   },
   "config": {
-    "ampServerUrl": "amp-togo-develop.stg.ampsite.net",
-    "ampServerPort": "443",
-    "ampServerProtocol": "https"
+    "ampServerUrl": "localhost",
```

Comment:
Please do not commit to server configuration section

Reply:
good catch

Reply:
done